apps/vmq_webhooks/priv/vmq_webhooks.schema

```diff
@@ -83,3 +83,14 @@
  [{datatype, string},
   hidden,
   {default, "    webhooks    Manage webhooks\n"}]}.
+
+%% @doc Configue the maximum number of HTTP connections maintained in the webhooks pool.
+{mapping, "vmq_webhooks.hackney_pool_max_connections", "vmq_webhooks.hackney_pool_max_connections",
+ [{datatype, integer},
+  {default, 100}]}.
+
+%% @doc Configue the timeout that we keep the connection alive in the 
+%% webhooks connection pool, expressed in milliseconds. 
+{mapping, "vmq_webhooks.hackney_pool_timeout", "vmq_webhooks.hackney_pool_timeout",
```

Comment:
I don't like exposing into the config that we're using hackney so I'd remove that prefix.

I think it a name like `idle_connection_timeout` or something like that would be better (timeout is too generic).

Reply:
Great!  I will do that, perhaps `pool_connection_timeout`

Reply:
sounds good.